how-to-build-mobile.md

```diff
@@ -0,0 +1,39 @@
+# Building Cryptopower Wallet for Mobile
+
+This readme assumes you have a working Android or iOS environment.
+
+## 1. Building for Android
+
+Note: You need to have [gogio](https://gioui.org/doc/install/android) to build.
```

Comment:
The link points to the android build instructions page which begins with a mention of android dependencies. The link text should probably say `[some dependencies, including gogio]` rather than just `[gogio]`.

Reply:
@dreacot.